src/lib/rest/httpRequestSend.cpp

```diff
@@ -143,7 +143,7 @@ static char* curlVersionGet(char* buf, int bufLen)
 * PARAMETERS
```

Comment:
An entry in the CHANGES_NEXT_RELEASE file should be added with regards to this PR. Suggestion:

```
-Fix: case-senstive header duplication (e.g. "Content-Type" and "Content-type") in custom notifications (#2893)
```

Reply:
Fixed in 59aea87